mentoring/tests/integration/test_mcq.py

```diff
@@ -231,3 +258,20 @@ def test_questionnaire_html_choices(self, page):
         self.wait_until_disabled(submit)
 
         self.assertIn('Congratulations!', messages.text)
+
+
+class MCQBlockBlankThemeTest(MCQBlockTest):
+    """
+    Test MRQ/MCQ questions without the LMS theme which is on by default.
+    """
+    def __init__(self, *args, **kwargs):
+        super(MCQBlockBlankThemeTest, self).__init__(*args, **kwargs)
+
+    def setUp(self, *args, **kwargs):
+        super(MCQBlockBlankThemeTest, self).setUp(*args, **kwargs)
+        self._original_get_theme = MentoringBlock.get_theme
+        MentoringBlock.get_theme = lambda self: {'package': None, 'locations': []}
```

Comment:
I had considered using `mock`, but I am not very familiar with it and couldn't figure out how to use it to patch an instance method from reading the docs. Your example code helps a lot. I couldn't get the decorator version to work like I wanted (it was trying to pass an extra parameter to the tests), but initializing the patcher object in the `setUp` method works great, thanks.